naga/src/front/wgsl/parse/directive/enable_extension.rs

```diff
@@ -89,6 +97,7 @@ impl EnableExtension {
     const MESH_SHADER: &'static str = "wgpu_mesh_shader";
     const RAY_QUERY: &'static str = "wgpu_ray_query";
     const RAY_QUERY_VERTEX_RETURN: &'static str = "wgpu_ray_query_vertex_return";
+    const EXPERIMENTAL_COOPERATIVE_MATRIX: &'static str = "experimental_cooperative_matrix";
```

Comment:
The naming convention here is `wgpu_*` so this should be:

```suggestion
    const COOPERATIVE_MATRIX: &'static str = "wgpu_cooperative_matrix";
```

Similar transforms needed for the other enable extension variable names. Note the feature itself in wgpu should stay as it is.